Assets/Scripts/Drone/DroneOnDewPickup.cs

```diff
@@ -0,0 +1,49 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class DroneOnDewPickup : MonoBehaviour
+{
+    public static DroneOnDewPickup instance;
+
+    /// <summary>
+    /// Makes the drone rotate in a around the y axis and display the island heart when jellyDew is picked up the first time
+    /// </summary>
+
+    public bool didtheThing { get; private set; } = false;
+    [SerializeField] Sprite islandHeartImage;
+    [SerializeField] float rotationSpeed;
+    bool currentlyRotating = false;
+    float counter = 0;
+
+    void Start()
+    {
+        instance = this;
+    }
+
+    void Update()
+    {
+        if (currentlyRotating)
+        {
+            transform.Rotate(0, -rotationSpeed * 0.5f * Time.deltaTime, 0);
```

Comment:
The drone's movement should be via its state machine, so we don't have multiple scripts moving the drone.

The merge conflicts on this are going to be a nightmare though. We can just leave this as-is I guess, and do it in a different task. Maybe just add a comment like TODO: Use the drone's state machine as the source of this movement.